packages/amplify_auth_cognito/test/amplify_auth_cognito_getCurrentUser_test.dart

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Given that this file is already in `amplfy_auth_cognito/test`, do you need to include `amplify_auth_cognito` in the filename _again_? -- general question around naming for any/all test files.

Reply:
No, I think we can make that change to all the files (but it's going to make this PR appear much larger since it will be interpreted as deleted/new files, so I'm going to wait).